### ISSUE TITLE
[Hunter > Trapping Bug] 

### ISSUE THREAD

**Issue by DatJetto** [RoDNet/Bugtracker-Infinium-4.0.6a#397]
Hola otra vez, esta vez vengo a reportar un bug de las trampas del cazador, 

http://es.wowhead.com/spell=34600
http://es.wowhead.com/spell=1499
http://es.wowhead.com/spell=13809
http://es.wowhead.com/spell=13813
http://es.wowhead.com/spell=13795

Cada trampa ha de permanecer en el suelo 60 sec, con un cooldown base de 30 sec cada trampa, compartiendo CD  las del mismo elemento, hasta aquí todo bien.

El problema que se encuentra es que cuando pones por ejemplo; Trampa de hielo / ya sea con lanzador o sin él / si la trampa de hielo no se ha activado y colocamos por ejemplo trampa de culebras o explosiva, la de hielo desaparece, es decir, algo hace que solo podamos tener 1 trampa activa a la vez, debiendo no haber restricciones, me explico, debería de poder poner Trampa de hielo / Congelante  junto a trapa con culebras o trampa de inmolación / explosiva y tener las 3 en el suelo, así mismo si usamos Readiness (http://es.wowhead.com/spell=23989)  podríamos tener 6 trampas activas.

**Comment by asfo** [comment]
http://old.wowhead.com/spell=23989 <--Aquí dice que solo elimina el Cooldown, se supone o tengo entendido yo, el límite de trampas es 3...Nada más...

De lo demás, espero le echen un ojo :).

**Comment by DatJetto** [comment]
No, a ver, simplemente NO existe límite de trampas activas, cada trampa dura 1 minuto en el suelo y a los 60 sec la trampa se desvanece, el Readiness lo que hace es volver a poner todas las facultades activadas, de manera que por ejemplo pongo ice trap, uso readiness y pongo otra ice trap, ambas son válidas y no han de contrarrestarse. 
El mismo ejemplo con Bengala, uso bengala, readiness y bengala y ambas bengalas permanecen en el suelo, no se elimina una por que halla otra del mismo tipo.
